lib/fastboot-info.js

```diff
@@ -30,6 +31,10 @@ FastBootInfo.prototype.extractCookies = function(request) {
   return {};
 };
 
+FastBootInfo.prototype.extractHost = function(request) {
+  return `${request.protocol}://${request.headers.host}`;
```

Comment:
I'm using the host header because it also contains the port, and instead of reassembling that data from the parts parsed from the header, I'm just using the compiled header. If there were subdomains, we'd have to reconcatenate them before assembling the host